projects/legacy/components/arrow/arrow.component.ts

```diff
@@ -15,7 +15,7 @@ import {
 import {TUI_ARROW_OPTIONS} from './arrow.options';
 
 /**
- * @deprecated: drop in v5.0
+ * @deprecated: drop in v5.0 use {@link TuiChevron}
```

Comment:
Should we add link to this example ?

https://taiga-ui.dev/components/textfield#dropdown